Add README.md's (#14)
sonjaleo/KinFragLib: data/filters/README.md

# Filters Data

> - `Brenk`: folder containing the 'unwanted_substructures.csv' provided by Brenk et al.

Reviewer: ```suggestion
- `Brenk`: Folder containing the 'unwanted_substructures.csv' provided by Brenk et al.
```

> - `DataWarrior`: folder containing the 'Enamine_Building_Blocks.sdf' file created with DataWarrior.

Reviewer: ```suggestion
- `DataWarrior`: Folder containing the 'Enamine_Building_Blocks.sdf' file created with DataWarrior.
```

> - `retrosynthesizability`: folder containing 'retro.txt' where all retrosynthetic pathways that were and will be requested are stored

Reviewer: ```suggestion
- `retrosynthesizability`: Folder containing 'retro.txt' in which all retrosynthetic pathways that were and will be requested are stored.
```

Reviewer: what means 'will be requested' here? Is this file not static?

Author: No, it's not static.
If one keeps more fragments in the previous filtering steps and new ASKCOS queries are started, the results will also be stored there.

Reviewer: ok, then change to: 
- `retrosynthesizability`: Folder containing 'retro.txt' in which all retrosynthetic pathways that were previously requested (or will potentially be assembled in future queries) are stored.
